src/Workspaces/MSBuildTest/Microsoft.CodeAnalysis.Workspaces.MSBuild.UnitTests.csproj

```diff
@@ -4,7 +4,7 @@
   <PropertyGroup>
     <OutputType>Library</OutputType>
     <RootNamespace>Microsoft.CodeAnalysis.MSBuild.UnitTests</RootNamespace>
-    <TargetFrameworks>net6.0-windows;net472</TargetFrameworks>
+    <TargetFrameworks>net8.0-windows;net472</TargetFrameworks>
```

Comment:
Was this upgrade just because we're testing templates that don't exist on 6.0? Can we have the theories just count themselves as skipped if the template doesn't exist?

Reply:
I believe it was due to the test runner running on .NET 6 runtime trying to load a .NET 8.0 project. Maybe we could leave as net6.0 if the build host is being used.

Reply:
No longer applicable.